README.md

```diff
@@ -1,3 +1,7 @@
+# Team Members
+
+Jillian (@catbus00) is a Code the Dream student studying React.
```

Comment:
sorry I gave you a bad suggestion last time, turns out that `@` tag doesn't parse inside the markdown file like it would in the github page itself

```suggestion
* **Jillian** \([@catbus00](https://github.com/catbus00)\) is a student studying React, but showing Backend Team some support!
```